src/Angelica/kata31_10/Factorial.py

```diff
@@ -0,0 +1,5 @@
+def factorial(number):
+    if number == 0:
```

Comment:
you can use ternary condition as following: `return 1 if number == 0 else number * factorial(number - 1)`